test-fixtures/update-rust-component-version.py

```diff
@@ -0,0 +1,117 @@
+import os, json
+import requests
+import datetime
+from github import Github
+
+
+GITHUB_REPO = "mozilla/rust-components-swift"
+SPM_PACKAGE = "Client.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved"
+FIREFOX_PROJECT = "Client.xcodeproj/project.pbxproj"
+github_access_token = os.getenv("GITHUB_TOKEN")
+
+def get_latest_rust_components_version():
+    g = Github(github_access_token)
+    repo = g.get_repo(GITHUB_REPO)
+
+    latest_tag = repo.get_tags()[0].name
+    latest_commit = str(repo.get_tags()[0].commit)
+    only_commit = re.findall(r'"([^"]*)"', latest_commit)
+
+    return (str(latest_tag), str(only_commit[0]))
+
+def read_rust_components_tag_version():
+    # Read Package file to find the current rust-component version
```

Comment:
Current or newest?

Reply:
Current as it is reading the rust-component version in project file